docs/internals/ingest-v2.md

```diff
@@ -18,7 +18,3 @@ indexer:
 See [full configuration example](https://github.com/quickwit-oss/quickwit/blob/main/config/quickwit.yaml).
 
 The only way to use the ingest API V2 is to use the [bulk endpoint](../reference/es_compatible_api.md#_bulk--batch-ingestion-endpoint) of the Elasticsearch-compatible API. The native Quickwit API is not yet compatible with the ingest V2 API.
-
-## Caveats
-
-The `refresh` parameter is not yet supported on the ingest V2 API.
```

Comment:
This has nothing to do with this PR but was left out of https://github.com/quickwit-oss/quickwit/pull/5350